atlp-rwanda/e-comm-team-techsmith-fn
# Design mockup design to define application theme

## What does this PR do?

This PR creates a mockup design to define the application for future references. Adding to this, it identifies all reusable components and their respective props.

## Description of Task to be completed?

+ Design the home, authentication, single product, and dashboard pages to define a global theme for the website
+ Identify reusable components and define their respective props
+ Define global colors

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout ft-design-mockup`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm run dev`

- Check out the components documentation: `npm run storybook`

## Any background context you want to provide?

This PR is not complete. It lacks the following features:
+ Responsiveness for multiple screens
+ The components are not yet documented on storybook

## Screenshots (if appropriate)

<img width="1440" alt="Screenshot 2023-05-25 at 17 15 33" src="https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/31816352/00c3eead-8132-439d-b561-aed5bdb7967f">

<img width="1440" alt="Screenshot 2023-05-26 at 16 06 56" src="https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/31816352/e6caa9e5-7efc-481a-8de6-1338592cb2b4">

<img width="1440" alt="Screenshot 2023-05-31 at 11 39 22" src="https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/31816352/9177adcf-849d-487e-a42a-821f1ab8359c">



https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/31816352/c86e52a8-e43b-4521-85cd-2a2bb3f8fc90
